tests/golos.vesting_tests.cpp

```diff
@@ -221,6 +221,27 @@ BOOST_FIXTURE_TEST_CASE(buy_vesting, golos_vesting_tester) try {
     CHECK_MATCHING_OBJECT(vest.get_stats(), mvo()("supply", vest.asset_str(buy1+buy2)));
 } FC_LOG_AND_RETHROW()
 
+BOOST_FIXTURE_TEST_CASE(bulk_buy_vesting, golos_vesting_tester) try {
+    BOOST_TEST_MESSAGE("Test bulk buying vesting / converting token to vesting");
+    BOOST_CHECK_EQUAL(success(), token.create(cfg::emission_name, token.make_asset(1000), {cfg::charge_name, cfg::publish_name}));
+    BOOST_CHECK_EQUAL(success(), token.issue(cfg::emission_name, N(sania), token.make_asset(500), "issue tokens sania"));
+    produce_block();
+
+    BOOST_CHECK_EQUAL(success(), vest.create_vesting(cfg::emission_name));
+    BOOST_CHECK_EQUAL(success(), vest.open(N(sania)));
+    BOOST_CHECK_EQUAL(success(), vest.open(N(pasha)));
+    BOOST_CHECK_EQUAL(success(), vest.open(N(tania)));
+    BOOST_CHECK_EQUAL(success(), vest.open(N(vania)));
+    produce_block();
+
+    BOOST_CHECK_EQUAL(success(), token.bulk_transfer( N(sania), {{cfg::vesting_name, token.make_asset(200), "buy vesting"},
```

Comment:
the actual bulktransfer to vesting works differently than this test.
please add case there different recipients are passed in memo fields (think of posting contract buying vesting to curators).

Reply:
This test will be implemented in the task #637

Reply:
your code already handles such transfers, but it have no tests.
payment to curators is just a sample. current implementation already allows anybody to buy vesting to several recipients passing they account names to memos. so we need this test to ensure it works.